external/external.go

```diff
@@ -0,0 +1,107 @@
+package external
```

Comment:
Can you rename this package to `exported` and the file to `exported.go`. Reason for saying this is to follow the same naming convention across Blobfuse and GO SDK.